GVFS/GVFS.UnitTests/Mock/FileSystem/MockFileSystemWithCallbacks.cs

```diff
@@ -48,11 +48,6 @@ public override void DeleteFile(string path)
         {
         }
 
-        public override void DeleteDirectory(string path, bool recursive = false)

```

Comment:
Can this method stay now that we're keeping `DeleteDirectory`?

Reply:
I've renamed `RecursiveDelete` to `DeleteDirectory(..., bool recursive)`. So we already have that (`DeleteDirectory`). The old `DeleteDirectory` is removed only becoz it didn't do anything more than redirecting to `RecursiveDelete`, I suggest we don't bring it back.

Reply:
Missed it, good catch... Added the override back.